### PR TITLE
OCPEDGE-1035: feat: Adding the AWS metal realtime lane for 4.17

### PR DESCRIPTION
**Pull request by jeff-roche**
This adds the performance profile and workflow config for running the real time kernel on AWS metal instances.

### CODE REVIEW

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-metal-ovn-single-node-rt

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-upgrade-from-stable-4.16-e2e-aws-upgrade-ovn-metal-single-node-rt

**Comment by jeff-roche** [comment]
The test failures are expected. Most of the failures are matching the SNO lanes and the workload partitioning failure is being worked separately by @eggfoobar

**Comment by jeff-roche** [comment]
/assign @vrutkovs 

Accidentally assigned Stephen out of habit

**Review by vrutkovs — APPROVED** [review]
/approve

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.17-upgrade-from-stable-4.16.yaml [hunk 1]
```diff
@@ -90,6 +90,17 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-upgrade-aws-single-node
+- as: e2e-aws-upgrade-ovn-metal-single-node-rt
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.17-upgrade-from-stable-4.16.yaml:93]: This is going to cause problems for how we report what this job is.  If it's metal, the prior art is to just say metal and not mention the underlying platform provider.

That lets us attribute the job results to the right place.

Suggest: e2e-metal-ovn-single-node-rt-upgrade

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-upgrade-from-stable-4.16-e2e-metal-ovn-single-node-rt-upgrade

**Pull request merged**